set.mm

```diff
@@ -325398,32 +325439,40 @@ or are almost disjoint (the interiors are disjoint).  (Contributed by
     ${
       mbfeqalem.4 $e |- ( ( ph /\ x e. B ) -> C e. RR ) $.
       mbfeqalem.5 $e |- ( ( ph /\ x e. B ) -> D e. RR ) $.
-      $( Lemma for ~ mbfeqa .  (Contributed by Mario Carneiro, 2-Sep-2014.) $)
+      $( Lemma for ~ mbfeqalem .  (Contributed by Mario Carneiro, 2-Sep-2014.)
+         (Revised by AV, 19-Aug-2022.) $)
+      mbfeqalemlem $p |- ( ph -> ( ( `' ( x e. B |-> C ) " y )
```

Comment:
Side note: labels for lemmas generally follow the naming xxxlem1, xxxlem2... instead of xxxlemlem..., even if a lemma is referencing another. But this is very minor, and does not even need to be changed IMO.
So: ready to merge.